assets/js/backend.js

```diff
@@ -116,5 +116,18 @@
 
 			e.preventDefault();
 		});
+
```

Comment:
Please re-use our click-to-copy JS code instead of duplicating it, if it works for readonly inputs only, it's ok if you also include textareas there.